opentech/apply/funds/templates/funds/applicationsubmission_detail.html

```diff
@@ -32,7 +32,7 @@ <h5 class="heading heading--meta">
                 <a class="tab__item" href="#activity-feed" data-tab="tab-3">
                     Activity Feed
                 </a>
-                {% if request.user.is_superuser %}
+                {% if request.user.is_staff %}
```

Comment:
[is_apply_staff](https://github.com/OpenTechFund/opentech.fund/blob/master/opentech/apply/users/models.py#L90) property seems better here.

Reply:
Good catch, have fixed that now.